### ISSUE TITLE
FIXED: Startup config check inconsistent with tool import requirements

### ISSUE THREAD

**Issue by rayketcham** [rayketcham-lab/vox#54]
## Problem (Fixed in v0.6.2)
The startup config summary in `pipeline.py` checked `import diffusers` (bare module), but the actual tool in `tools.py` imports `import torch; from diffusers import StableDiffusionPipeline`. This meant the startup could show "image-gen" as available even when torch was missing, or NOT show it when diffusers was installed but StableDiffusionPipeline import failed.

## Fix
Changed `pipeline.py` to check the same imports the tool uses:
```python
import torch
from diffusers import StableDiffusionPipeline
```

## Lesson Learned
Feature detection checks should match the actual import path used by the feature, not a simplified version.

## Version
Fixed in v0.6.2 (commit 8bcf7f1)

## Labels
bug, resolved
